Library/Homebrew/blacklist.rb

```diff
@@ -1,16 +1,5 @@
 def blacklisted? name
   case name.downcase
-  when 'screen', /^rubygems?$/ then <<-EOS.undent
```

Comment:
Also I'm a little hesitant on removing formulae which are available by taps. Blacklist is used to remind users that certain tools are already available in OS X, while dupes tap is used to offer alternate newer version of these tools. Since blacklist message will be auto ignored when formulae exist, I see no harm to keep them here.